packages/react-components/react-spinbutton/src/stories/SpinButtonAppearance.stories.tsx

```diff
@@ -34,25 +43,41 @@ export const Appearance = () => {
   const filledDarkerId = useId('filledDarker-id');
 
   return (
-    <div className={styles.root}>
+    <div className={styles.base}>
       <div>
         <Label htmlFor={outlineId}>Outline (default)</Label>
-        <SpinButton id={outlineId} />
+        <div>
+          <SpinButton id={outlineId} />
+        </div>
       </div>
 
       <div>
         <Label htmlFor={underlineId}>Underline</Label>
-        <SpinButton appearance="underline" id={underlineId} />
+        <div>
+          <SpinButton appearance="underline" id={underlineId} />
+        </div>
       </div>
 
-      <div className={styles.filledLighter}>
+      <div>
         <Label htmlFor={filledLighterId}>Filled Lighter</Label>
-        <SpinButton appearance="filledLighter" id={filledLighterId} />
+        <div className={styles.filledLighter}>
+          <SpinButton appearance="filledLighter" id={filledLighterId} />
+        </div>
       </div>
 
-      <div className={styles.filledDarker}>
+      <div>
         <Label htmlFor={filledDarkerId}>Filled Darker</Label>
-        <SpinButton appearance="filledDarker" id={filledDarkerId} />
+        <div className={styles.filledDarker}>
+          <SpinButton appearance="filledDarker" id={filledDarkerId} />
+        </div>
+      </div>
+
+      <div>
+        <Text>
+          The colors adjacent to the input should have a sufficient contrast. Particularly, the color of input with
+          filled darker and lighter styles needs to provide greater than 3 to 1 contrast ratio against the immediate
+          surrounding color to pass accessibility requirements.
```

Comment:
This should be in the story description below, not text within the story itself.